docker/test/integration/minifi/core/DockerTestCluster.py

```diff
@@ -54,13 +55,16 @@ def get_app_log(self, container_name):
 
         return container.status, None
 
-    def wait_for_app_logs(self, container_name, log_entry, timeout_seconds, count=1):
+    def wait_for_app_logs(self, container_name, log_entry, timeout_seconds, count=1, use_regex=False):
```

Comment:
I would prefer having a separate function for log matching and regex matching, the surrounding code could may be extracted as a decorator.